src/torcontrol.cpp

```diff
@@ -53,6 +53,9 @@ const std::string DEFAULT_TOR_CONTROL = "127.0.0.1:" + ToString(DEFAULT_TOR_CONT
 static const int TOR_COOKIE_SIZE = 32;
 /** Size of client/server nonce for SAFECOOKIE */
 static const int TOR_NONCE_SIZE = 32;
+//* Tor control reply code. Ref: https://spec.torproject.org/control-spec/replies.html */
```

Comment:
nit: Please use the same comment syntax as the other comments

Reply:
Thank you, fixed.